inst/rmarkdown/templates/ajs/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,221 @@
+---
+author:
+  - name: FirstName LastName
+    affiliation: University/Company
+    # use this syntax to add text on several lines
+    address: |
+      | First line
+      | Second line
+    email: \email{name@company.com}
+    url: http://rstudio.com
+  - name: Second Author
+    affiliation: 'Affiliation \AND'
+    # To add another line, use \AND at the end of the previous one as above
+  - name: Third Author
+    address: |
+      | Department of Statistics and Mathematics,
+      | Faculty of Biosciences,
+      | Universitat Autònoma de Barcelona
+    affiliation: |
+      | Universitat Autònoma 
+      | de Barcelona
+    # use a different affiliation in adress field (differently formated here)
+    affiliation2: Universitat Autònoma de Barcelona
+title:
+  formatted: "A Capitalized Title: Something about a Topic"
+  # If you use tex in the formatted title, also supply version without
+  plain:     "A Capitalized Title: Something about a Topic"
+  # For running headers, if needed
+  short:     "A Capitalized Title"
+abstract: >
+  This guide of the Austrian Journal of Statistics (AJS) shows the required style of manuscripts submitted to the Austrian Journal of Statistics. We strongly recommend to follow the given instructions exactly already when writing the paper. When submitting papers to AJS, it is essential to follow these guidelines. Manuscripts submitted without consideration of the AJS style guide will most likely not be forwarded for review, but rejected with the possibility of resubmission.
+keywords:
+  # at least one keyword must be supplied
+  formatted: [keywords, not capitalized, "\\proglang{Java}"]
+  plain:     [keywords, not capitalized, Java]
+preamble: >
+  \usepackage{amsmath}
```

Comment:
Would you be ok here to switch to 

```suggestion
header-includes:
 - \usepackage{amsmath}
```

This will work with the template and is the more standard way to do that with Pandoc.

Reply:
Ok!